Improvements to module 102 (#2)
MicrosoftLearning/AZ-030T00-Microsoft-Azure-technologies-for-AWS-architects: Instructions/Mini-labs/Module 01/02-MiniLab_Quickstart_for_Bash_in_Azure_Cloud_Shell​.md

1. Launch **Cloud Shell** from the top navigation of the Azure portal.

    ![Cloud Shell icon](../../Linked_Image_Files/shell-icon.png)
    

>     * If you don't see the Shell icon, it may be under the **three-point icon**
>         ![Three points icon](../../Linked_Image_Files/three-points.png)

Reviewer: ellipses

```suggestion
    * If you don't see the Shell icon, it may be under the **ellipses icon**
        ![Ellipses icon](../../Linked_Image_Files/three-points.png)
```

    ```
    az account list
    ```
    * Note the 'name' of the subscription 

1. Set your preferred subscription:

    ```
    az account set --subscription 'my-subscription-name'
    ```
    * Replace 'my-subscription-name' with the name of a subscription you like
    * This will change the subscription that you are working on. If you want to undo this change, simply write the following command and replace 'old-subscription-name' with the name of the subscription you want to keep using

        ```
        az account set --subscription 'old-subscription-name'
        ```

>     > If you need more information on subscriptions, click [here](https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/4-multiple-subscriptions)

Reviewer: since this is a guided in class lab will remove the references to docs

```suggestion

```

>:pencil: **Tip:** Your subscription will be remembered for future sessions in `/home/<user>/.azure/azureProfile.json`.

## Create a resource group

> 1. Create a new resource group in WestUS named "MyRG".

Reviewer: EastUS

```suggestion
1. Create a new resource group in EastUS named "MyRG".
```

    ```
    az group create --location eastus --name MyRG
    ```

>    > If you need more information on resource groups, click [here](https://docs.microsoft.com/en-us/learn/modules/control-and-organize-with-azure-resource-manager/2-principles-of-resource-groups)

Reviewer: remove documentation link

```suggestion

```

1. Search for your VM name in the Azure portal search bar.

> 1. Click on the VM named **MyVM**

Reviewer: capitalization removed
```suggestion
1. Click on the VM named **myVM**
```

1. Click **Connect** to get your VM name and public IP address.

    ![Connect option](../../Linked_Image_Files/sshcmd-copy.png)

> 1. Under the **Connect via SSH with client**, find the 4th point **"Run the example command below to connect to your VM"**

Reviewer: Azure UI constantly changes so we will be less precise
```suggestion
1. Under the **Connect via SSH with client**, select **"Run the example command below to connect to your VM"**
```

> 1. Notice the command it shows:

Reviewer: edit 

```suggestion
1. Notice the command:
```
